apps/src/Sound.js

```diff
@@ -339,22 +340,55 @@ Sound.prototype.getPlayableFile = function() {
   return false;
 };
 
+Sound.prototype.getPlayableBytes = function() {
+  try {
+    if (!window.Audio) {
+      return false;
```

Comment:
why `false` not `undefined` or `null`?

Reply:
I was following the style of getPlayableFile (the function above). Should I switch to undefined?